storehaus-http/src/main/scala-2.10/com/twitter/storehaus/http/compat/package.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 Twitter Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twitter.storehaus.http
+
+package object compat {
```

Comment:
Again, not sure how important it is to support 2.10. If not then looks like we can avoid this compat code.